src/components/videos/index.js

```diff
@@ -27,7 +32,7 @@ export function Vidoes() {
             }
             dispatch({ type: 'LOADING_VIDEOS', payload: false });
         })()
-    // eslint-disable-next-line react-hooks/exhaustive-deps
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
suggestion: try to find out any other solution to this problem as it's not a good practice to add this comment